src/main/proto/spine/options.proto

```diff
@@ -127,24 +127,32 @@ extend google.protobuf.FieldOptions {
     // See `PatternOption`.
     PatternOption pattern = 73820;
 
-    // Enables validation constraint checking for fields that refer to a message.
+    // Enables in-depth validation for fields that refer to a message.
     //
-    // Default value: `false`.
+    // This option applies only to fields that reference a message:
     //
-    // This option can only be applied to fields that refer to a message. Such fields include:
+    //   1. Singular message fields.
+    //   2. Repeated fields of message types.
+    //   3. Map fields with message types as values.
     //
-    //   1. Message fields.
-    //   2. Map fields with message types as values.
-    //   3. Repeated fields of message types.
+    // When set to `true`, the field is valid only if its value satisfies the validation
+    // constraints defined in the corresponding message type:
     //
-    // When set to `true`, the marked field is considered valid only if the field's value satisfies
-    // the validation constraints defined in the corresponding message type. This means:
+    //   1. For singular message fields: the message must meet its constraints.
+    //      Note: default instances are considered valid even if the message has required fields.
```

Comment:
Please have examples for each of these points.

For this particular point, it would be helpful to illustrate the difference between 
```
MyMessage msg = 42 [(validate) = true]; // implicit `(required) = false`.
```
and
```
MyMessage msg = 42 [(validate) = true, (required) = true];
```